service/src/main/java/com/zebrunner/reporting/service/TestRunService.java

```diff
@@ -249,6 +250,11 @@ public TestRun getTestRunByCiRunIdFull(String ciRunId) {
         return automationServerService.getBuildConsoleOutput(testRun.getJob(), testRun.getBuildNumber(), count, fullCount);
     }
 
+    @Transactional(readOnly = true)
+    public List<TestRunResult> getTestRunResultsByTestSuiteId(Long testSuiteId, Long limit) {
+        return testRunMapper.getTestRunResultsByTestSuiteId(testSuiteId, limit);
```

Comment:
How many db queries will be performed by this call? If only one, then it is much better to remove `@Transactional`.